### PR TITLE
CLOUDTRUST-1425 Paginate users and return total count

### PR DESCRIPTION
**Pull request by Fratt**
**Warning**
This needs to be merged simultaneously with following PR to handle the new format of the response!
* cloudtrust/keycloak-client#14
* cloudtrust/keycloak-bridge#94

### CODE REVIEW

**Comment by Fratt** [comment]
Currently updating the unit tests, don't merge!

File: src/main/java/io/cloudtrust/keycloak/representations/idm/UsersPageRepresentation.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+     */
+    List<UserRepresentation> users;
+
+    /**
+     * Total users count for those search criteria
+     */
+    int count;
+
+    /**
+     * For unit tests
```
Comment (fperot74) [inline src/main/java/io/cloudtrust/keycloak/representations/idm/UsersPageRepresentation.java]: `For unserializing` would be better

File: src/main/java/io/cloudtrust/keycloak/representations/idm/UsersPageRepresentation.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+public class UsersPageRepresentation {
+
+    /**
+     * Users
+     */
+    List<UserRepresentation> users;
+
+    /**
+     * Total users count for those search criteria
```
Comment (fperot74) [inline src/main/java/io/cloudtrust/keycloak/representations/idm/UsersPageRepresentation.java]: There is no mention of any search criteria here... I would simplify as `Total users count`

File: src/main/java/io/cloudtrust/keycloak/services/resource/api/admin/UsersResource.java [hunk 3]
```diff
@@ -1,5 +1,6 @@
 package io.cloudtrust.keycloak.services.resource.api.admin;
 
+import io.cloudtrust.keycloak.representations.idm.UsersPageRepresentation;
 import io.cloudtrust.keycloak.services.NotImplementedException;
```
Comment (fperot74) [inline src/main/java/io/cloudtrust/keycloak/services/resource/api/admin/UsersResource.java]: You can remove this line: NotImplementedException is not used anymore

Comment (Fratt) [reply src/main/java/io/cloudtrust/keycloak/services/resource/api/admin/UsersResource.java]: Well fine, then I ran an "Optimize imports" on the whole thing, there were other ghost imports :-)

File: src/main/java/io/cloudtrust/keycloak/representations/idm/UsersPageRepresentation.java (hunk 2 above)
Comment (Fratt) [reply src/main/java/io/cloudtrust/keycloak/representations/idm/UsersPageRepresentation.java]: Fixed

File: src/main/java/io/cloudtrust/keycloak/representations/idm/UsersPageRepresentation.java (hunk 1 above)
Comment (Fratt) [reply src/main/java/io/cloudtrust/keycloak/representations/idm/UsersPageRepresentation.java]: Fixed

File: src/main/java/io/cloudtrust/keycloak/services/resource/api/admin/UsersResource.java (hunk 3 above)
Comment (fperot74) [reply src/main/java/io/cloudtrust/keycloak/services/resource/api/admin/UsersResource.java]: There are lots of them in many projects ! ;-)

**Pull request merged**